torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2913,6 +2913,28 @@ def sample_inputs_diag(op_info, device, dtype, requires_grad, **kwargs):
 
     return samples + [vec_sample]
 
+def sample_inputs_diagonal(op_info, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, dtype=dtype, device=device, requires_grad=requires_grad, low=None, high=None)
```

Comment:
https://github.com/pytorch/pytorch/blob/5caccbe39e912281bff7cd372c7376fdca56905f/torch/testing/_internal/common_utils.py#L1725

You can skip passing `low` and `high`

Reply:
Yes, done, thanks!